selfdrive/car/toyota/interface.py

```diff
@@ -268,7 +268,12 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     ret.communityFeature = ret.enableGasInterceptor or ret.enableDsu or smartDsu
 
     if ret.enableGasInterceptor:
-      set_long_tune(ret.longitudinalTuning, LongTunes.PEDAL)
+      # these cars have full speed DRCC, use stock tune
+      if candidate in [CAR.PRIUS, CAR.RAV4H, CAR.HIGHLANDER, CAR.HIGHLANDERH, CAR.LEXUS_CTH, CAR.LEXUS_ESH,
+                       CAR.LEXUS_NX, CAR.LEXUS_NXH, CAR.LEXUS_RX, CAR.LEXUS_RXH]:
```

Comment:
put this list in values.py

Reply:
Is the RAV4 hybrid a no stop timer car? Does the driver need to press the resume button for it to resume?

Reply:
* asking because the RAV4H is mentioned in NO_STOP_TIMER_CAR, but it's also a pedal car in the new pedal code https://github.com/commaai/openpilot/blob/9ceb1a15eb87ba9ea5d7c8dbcc176f2466c82338/selfdrive/car/toyota/carcontroller.py#L32